book/website/ethical-research/activism/activism-digipres.md

```diff
@@ -0,0 +1,49 @@
+(er-activism-digipres)=
+# Guerrilla Digital Preservation
+
+[Management and preservation of data, code and other artefacts](#rr-rdm) used for a research project is essential. While there may be constraints on the time and other resources needed to do this, under ideal circumstances there will not be a person or natural disaster actively working against you, but not all research can take place in ideal circumstances.
+
+In 1933, the library and archives of the Institut für Sexualwissenschaft were looted and ultimately burned by members of the Deutsche Studentschaft and Sturmabteilung for being "un-German". The first institute of its kind in the world, the Institut conducted world-leading research into issues around gender and sexuality, and its destruction set back women's and LGBTQ+ right back decades. In September 2018 the National Museum of Brazil's air conditioning overheated due to a fault, resulting in a catastrophic fire and the destruction of approximately 92.5% of the Museum's 20 million-item archive.
```

Comment:
I think it would make sense to provide links so that folks can read more about both of these events.

Perhaps:
- https://en.wikipedia.org/wiki/Institut_f%C3%BCr_Sexualwissenschaft#Raids_and_book_burnings
- https://en.wikipedia.org/wiki/National_Museum_of_Brazil_fire